chart/skywalking/templates/oap-deployment.yaml

```diff
@@ -96,17 +96,26 @@ spec:
 {{ else }}
           tcpSocket:
             port: 12800
-          initialDelaySeconds: 15
-          periodSeconds: 20
+          initialDelaySeconds: 5
+          periodSeconds: 10
+{{ end }}
+        startupProbe:
+{{- if .Values.oap.startupProbe}}
+{{ toYaml .Values.oap.startupProbe | indent 10 }}
+{{ else }}
+          tcpSocket:
+            port: 12800
+          failureThreshold: 9
+          periodSeconds: 10
```

Comment:
What do you think if we remove this (Line 106 ~ Line 109`) and uncomment the `startupProbe` section in `values.yaml`?